contracts/EvitaToken.sol

```diff
@@ -11,10 +11,13 @@ contract EvitaToken is ERC20BurnableUpgradeable, OwnableUpgradeable {
     string public symbol_;
     uint256 public allSupply;
 
-    // todo in 2: move to init
-    constructor(uint256 _initialSupply) /* ERC20BurnableUpgradeable("Evita", "EVI") */ {
-        name_ = "Evita";
-        symbol_ = "EVI";
+    IERC20Upgradeable public token;
```

Comment:
Link to token must be in proxy. According to your test you deploy two tokens and pass one of them as param to the another.

So the main idea - we have (just) ordinary token (initialize instead of constructor). This deployed token is not used, but we use proxy which points to this token code